src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -1918,7 +1934,7 @@ char* os::pd_attempt_reserve_memory_at(char* requested_addr, size_t bytes) {
 
   // Bsd mmap allows caller to pass an address as hint; give it a try first,
   // if kernel honors the hint then we can return immediately.
-  char * addr = anon_mmap(requested_addr, bytes);
+  char * addr = anon_mmap(requested_addr, bytes, false/*executable*/);
```

Comment:
use `!ExecMem`?

Reply:
Agree, fixed. I've avoided that, as formally it's for `os::` interface layer. But unlikely it's worth even a bit of readability